images/macos/provision/core/ruby.sh

```diff
@@ -4,3 +4,36 @@ source ~/utils/utils.sh
 
 echo Installing Ruby...
 brew install ruby
+
+echo "Install Ruby..."
+PACKAGE_TAR_NAMES=$(curl -s "https://api.github.com/repos/ruby/ruby-builder/releases/latest" | jq -r '.assets[].name')
+TOOLSET_VERSIONS=$(get_toolset_value '.toolcache[] | select(.name | contains("Ruby")) | .versions[]')
+TOOLSET_PLATFORM=$(get_toolset_value '.toolcache[] | select(.name | contains("Ruby")) | .platform')
```

Comment:
Do we need to read it from toolset? I guess `platform` is identical for all macOS versions.

Reply:
Thank you. Not really. I removed `platform`  from toolset.